test/Package.js

```diff
@@ -24,7 +24,7 @@ describe("Package", () => {
         scripts: { "my-script": "echo 'hello world'" },
         dependencies: { "my-dependency": "^1.0.0" },
         devDependencies: { "my-dev-dependency": "^1.0.0" },
-        peerDependencies: { "my-peer-dependency": "^1.0.0" }
+        peerDependencies: { "my-peer-dependency": ">=1.0.0" }
```

Comment:
um, this isn't a good change; "as broad as possible" requires `^`, because declaring compatibility with `^1` is not the same thing as declaring infinite future compatibility across future semver-major lines.

Can this be fixed in a followup?

Reply:
This is just to make very _very_ explicit that we are never using `peerDependencies` for version "pairing" (that is, identifying lerna-managed sibling dependencies by name + semver) in this test.

The relevant test is here: https://github.com/evocateur/lerna/blob/5f38efe0227578757fb64f02a4a891468047f408/test/PublishCommand.js#L145-L148

Lerna never ever ever ever touches `peerDependencies` now, not even to compute sibling relationships. The `Package` getter is just leftover because I was too squeamish about removing it completely (with the idea that we would eventually reuse it for the improved experienced described in #1018).

Reply:
ok cool, i was worried that this indicated that lerna actually *changed* `^` to `>=`. Never touching peer deps is better than doing the wrong thing, that's for sure (but not as good as doing the right thing, which is keeping the existing semver range but widening it as needed to include the new version)